robot/packages/cloud_package_manager.go

```diff
@@ -545,7 +545,7 @@ func (m *cloudManager) unpackFile(ctx context.Context, fromFile, toDir string) e
 			// but whose files names start with a new directory prefix
 			// Ex: tar -czf package.tar.gz ./bin/module.exe
 			parent := filepath.Dir(path)
-			if err := os.MkdirAll(parent, info.Mode()); err != nil {
+			if err := os.MkdirAll(parent, 0o700|info.Mode()); err != nil {
```

Comment:
Good catch. Clearly a typo.

However, In retrospect, I'm not sure why the parent dir's permissions would relate to the child file's perms.

I suspect this should actually just be a constant `0o755`

Reply:
going to use 0775 -- it's the default for mkdir on linux and it's what the `tar` tool does when unpacking